vic-bunny/gulpfile.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const webpack = require('webpack-stream');
+
+var files = ['*.js', './app/*.js', './app/js/*.js', './app/js/photoalbum/*.js', './app/js/photoalbum/controllers/*.js', './app/js/photoalbum/directives/*.js'];
+
+const paths = {
+  js:__dirname + '/app/js/**/**/*.js',
+  html:__dirname + '/app/**/*.html',
+  css:__dirname + '/app/css/*.css'
+};
+
+gulp.task('lint', () => {
+  return gulp.src(files)
+  .pipe(eslint())
+  .pipe(eslint.format());
+});
+
+gulp.task('copy-html', () => {
+  return gulp.src(paths.html)
+  .pipe(gulp.dest(__dirname + '/build'));
+});
+
+gulp.task('copy-css', () => {
+  return gulp.src(paths.css)
+  .pipe(gulp.dest(__dirname + '/build'));
+});
+
+gulp.task('bundle', () => {
+  return gulp.src(paths.js)
```

Comment:
This will work.  But, since you have a client.js set up you should be using that to require in the rest if the js files.